string frame
time time_stamp
string fsm
bool good2fly
vector3D position
vector3D velocity
vector3D acceleration
vector3D jerk
vector4D orientation

# geometry_msgs::PoseStamped test
